### PR TITLE
test11111

### PR DESCRIPTION
**Pull request by NaiqianGuan**
test22222

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (NaiqianGuan) [inline test:1]: qqqqq

**Review by xiaominxie6 — APPROVED** [review]
good

**Review by xiaominxie6 — APPROVED** [review]
Suggestion: You could add some description about dataset and add the words below in the "Tidy data" part--- "According to the dataset6 with 2847 films, we choose the variables of year, length, budget, votes and genre as explanatory variables to analyse how these variables affect ratings of films."

**Review by xiaominxie6 — COMMENTED** [review]
Suggestion: You could add some description about dataset and add the words below in the "Tidy data" part--- "According to the dataset6 with 2847 films, we choose the variables of year, length, budget, votes and genre as explanatory variables to analyse how these variables affect ratings of films."

**Pull request closed without merging**